gradle-client/src/jvmMain/kotlin/org/gradle/client/ui/connected/actions/GetDeclarativeDocuments.kt

```diff
@@ -136,42 +142,19 @@ class GetDeclarativeDocuments : GetModelAction.GetCompositeModelAction<ResolvedD
             horizontallyScrollable = true,
             left = {
                 if (domWithDefaults != null) {
-                    val projectEvaluationSchema = projectResult.stepResults.values.single()
-                        .stepResultOrPartialResult.evaluationSchema
-
-                    val projectAnalysisSchema = projectEvaluationSchema.analysisSchema
-
-                    val mutationApplicability =
-                        MutationUtils.checkApplicabilityForOverlay(projectAnalysisSchema, domWithDefaults)
-
-                    val highlightingContext = HighlightingContext(
-                        domWithDefaults.overlayNodeOriginContainer,
-                        highlightedSourceRangeByFileId
-                    )
 
                     val softwareTypeNode = domWithDefaults.document.singleSoftwareTypeNode
                     if (softwareTypeNode == null) {
-                        Text("No software type")
+                        SettingsModelContent(highlightedSourceRangeByFileId, selectedDocumentResult)
```

Comment:
❓ Wouldn't `softwareTypeNode` be null also for an empty `build.gradle.dcl` file?

Reply:
Ah, you're right. 

It would make some sense to show the top level model for empty projects, too, as that would list the available software types, but that would not be so pretty (it would also say "Step: project" and for each software type, list its top level model's properties). Let's bring back the stub and maybe improve the empty project view at some point later.

I've fixed it in [`23d946a` (#29)](https://github.com/gradle/gradle-client/pull/29/commits/23d946a9309797ecb1ba76e8ebb382ad7f40a38f)